Week2/week02.js

```diff
@@ -0,0 +1,23 @@
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world',
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+connection.connect();
+
+const queries = [
+  'SELECT name FROM city WHERE id IN(SELECT capital FROM country WHERE name = ?);',
+  'SELECT distinct language FROM countrylanguage WHERE countrycode IN (SELECT code FROM country WHERE region = ?);',
+  'SELECT COUNT(name) AS "total cities" FROM city WHERE countrycode IN(SELECT countrycode FROM countrylanguage WHERE language = ?);',
+  'SELECT name FROM country WHERE region = ? AND code IN(SELECT countrycode FROM countrylanguage WHERE language = ? AND isofficial = "T");',
+  'SELECT name FROM country WHERE region = ? AND code IN(SELECT countrycode FROM countrylanguage WHERE language = ? AND isofficial = "T");',

```

Comment:
This line is the same as above ?

Reply:
yep :D that is my last minute hackjob, I was trying to create separate endpoint on node and it got complicated super fast so I just copied and pasted everything in one array and kinda slipped on the last one :D Will add the final one asap

As a gaming nerd I have a habit of keeping my fingers on (W, A, S, D) as some some sort of "combat stance" , after starting to code that stance has shifted to ( Ctrl, S, X, C, V), generally when I wear down keyboards it is always the (A) button first but now my (S) button is approaching the end of its lifespan on a brand new laptop keyboard :D